### DISCUSSION TITLE
Loophole: Multiple Linked Accounts should count as more than 1 free account used

### DISCUSSION THREAD

**Discussion by BenjaminWang236** [bkaankose/Wino-Mail#467] [Ideas]
TLDR: Title

Hello, thank you so much for this app. Currently switching to Wino Mail because MS Mail and Calendar kept re-installing "Outlook (new)" like a virus.

Anyways, I was setting up the Linked Accounts as I was adding each accounts, fully intending to upgrade to Unlimited Accounts midway through, when I noticed that linked account, which currently has 3 linked emails, counted as "1 out of 3 free accounts used". Seems like a loophole.

![ApplicationFrameHost_fI9S9OTgbf](https://github.com/user-attachments/assets/755f5fcd-01a5-4bd7-a1a4-7028c406a4fc)

**Comment by bkaankose** [comment]
Thanks for the report. Created an issue for it. Let's track this item there. #468
